numpy/ma/mrecords.py

```diff
@@ -28,7 +28,7 @@
 _byteorderconv = np.core.records._byteorderconv
 
 import numpy.ma as ma
-from numpy.ma import (
+from . import (
```

Comment:
This change sort of hides the fact that these two are importing from the same module.

Reply:
Could we remove line 30 and add the needed imports to the ones on line 31? The only tricky one is `ma.array`, which should probably be something like `from . import array as ma_array` to avoid confusion

Reply:
Perhaps doing the reverse would be clearer, not sure